ios/Runner/AppAuthHandler.swift

```diff
@@ -83,15 +109,15 @@ class AppAuthHandler {
             }
             
             guard let tokenResponse = authState.lastTokenResponse else {
-                self.invokeErrorHandler(errorMessage: "No token received")
+                self.invokeErrorHandler(code: FAILED_TOKEN_FETCH, errorMessage: "No token received")
                 return
             }
             
             // Request fresh token
             authState.performAction { (accessToken, idToken, error) in
                 
                 if let error = error {
-                    self.invokeErrorHandler(errorMessage: error.localizedDescription)
+                    self.invokeErrorHandler(code: self.FAILED_TOKEN_FETCH, errorMessage: error.localizedDescription)
```

Comment:
Is the `self` necessary here? Ditto all the others or missing above.

Reply:
It's needed for closures to reference method explicit.

Reply:
But for the constant?

Reply:
Thats swift.. 
<img width="834" alt="Bildschirmfoto 2020-11-07 um 22 30 27" src="https://user-images.githubusercontent.com/501326/98451764-27438e00-2149-11eb-991e-e80b451f2d88.png">